addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/EnvFileScanRule.java

```diff
@@ -38,8 +51,19 @@ public int getId() {
 
     @Override
     public boolean isFalsePositive(HttpMessage msg) {
+        HttpResponseHeader responseHeader = msg.getResponseHeader();
+        String contentType = responseHeader.getNormalisedContentTypeValue();
```

Comment:
This should be done later, when actually needed.

Reply:
moved